tools/licenses/lib/paths.dart

```diff
@@ -0,0 +1,385 @@
+  r'third_party/skia/third_party/icu', // we use our own version
+  r'third_party/skia/third_party/libjpeg-turbo', // we use our own version
+  r'third_party/skia/third_party/libpng', // we use our own version
+  r'third_party/skia/third_party/lua', // not linked in
+  r'third_party/skia/tools', // contains nothing that ends up in the binary executable
+  r'third_party/swiftshader', // only used on hosts for tests
+  r'third_party/tinygltf',
+  r'third_party/vulkan-deps/glslang/LICENSE', // excluded to make sure we don't accidentally apply it as a default license
+  r'third_party/vulkan-deps/glslang/src/LICENSE.txt', // redundant with licenses inside files
+  r'third_party/vulkan-deps/glslang/src/glslang/OSDependent/Web', // we only use glslang in impellerc, not in web apps
+  r'third_party/vulkan-deps/spirv-cross/src/LICENSES', // directory with license templates
+  r'third_party/vulkan-deps/spirv-cross/src/shaders', // used by regression tests
+  r'third_party/vulkan-deps/spirv-cross/src/shaders-hlsl', // used by regression tests
+  r'third_party/vulkan-deps/spirv-cross/src/shaders-hlsl-no-opt', // used by regression tests
+  r'third_party/vulkan-deps/spirv-cross/src/shaders-msl', // used by regression tests
+  r'third_party/vulkan-deps/spirv-cross/src/shaders-msl-no-opt', // used by regression tests
+  r'third_party/vulkan-deps/spirv-cross/src/shaders-no-opt', // used by regression tests
+  r'third_party/vulkan-deps/spirv-cross/src/shaders-other', // used by regression tests
+  r'third_party/vulkan-deps/spirv-cross/src/shaders-reflection', // used by regression tests
+  r'third_party/vulkan-deps/spirv-cross/src/shaders-ue4', // used by regression tests
+  r'third_party/vulkan-deps/spirv-cross/src/shaders-ue4-no-opt', // used by regression tests
+  r'third_party/vulkan-deps/spirv-headers', // only used on hosts for tests
+  r'third_party/vulkan-deps/spirv-tools', // only used on hosts for tests
+  r'third_party/vulkan-deps/vulkan-headers/src/registry',
+  r'third_party/vulkan-deps/vulkan-loader', // on hosts for tests
+  r'third_party/vulkan-deps/vulkan-tools', // on hosts for tests
+  r'third_party/vulkan-deps/vulkan-validation-layers/src/docs',
+  r'third_party/vulkan_memory_allocator/bin',
+  r'third_party/vulkan_memory_allocator/docs',
+  r'third_party/vulkan_memory_allocator/media',
+  r'third_party/vulkan_memory_allocator/src',
+  r'third_party/vulkan_memory_allocator/tools',
+  r'third_party/web_dependencies/canvaskit', // redundant; covered by Skia dependencies
+  r'third_party/wuffs/docs',
+  r'third_party/wuffs/script',
+  r'third_party/yapf', // only used for code formatting
+  r'third_party/zlib/contrib/minizip/miniunz.c', // sample file
+  r'third_party/zlib/contrib/minizip/minizip.c', // sample file
+  r'tools', // not distributed in binary
+};
+
+final Set<String> skippedCommonDirectories = <String>{
+  // directories that should be skipped anywhere we find them
```

Comment:
Is there a reason you are commenting these inside instead of above? Usually I expect comments describing the whole variable to be the line above, and comments inside the collection to group specific elements.

Reply:
no particular reason, switched to the other style